sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryStorageStreamSource.java

```diff
@@ -178,6 +179,7 @@ private BigQueryStorageStreamReader(
       this.parseFn = source.parseFn;
       this.storageClient = source.bqServices.getStorageClient(options);
       this.tableSchema = fromJsonString(source.jsonTableSchema, TableSchema.class);
+      this.splitPossible = true;
```

Comment:
Nit: I would just initialize this in the declaration above (e.g. ```private boolean splitPossible = true;```) rather than in the constructor.

Reply:
Done.